grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java

```diff
@@ -192,8 +191,9 @@ public static void addLibraryPath(String pathToAdd) throws Exception {
             if (path.equals(pathToAdd))
                 return;
 
-        String[] newPaths = Arrays.copyOf(paths, paths.length + 1);
-        newPaths[newPaths.length - 1] = pathToAdd;
+        String[] newPaths = new String[paths.length + 1];
```

Comment:
adding libraries to the front instead. as an added effect it will then prefer `jep` from the virtual env rather than the one in grobid-home.